README.md

```diff
@@ -950,7 +950,37 @@ Do not call this API with the service-operator-access plan credentials.
                                 
 <b>Attention: **Use this option only for cleanup purposes for a cluster that's no longer available.** Applying it to an active and available cluster may result in unintended resource leftovers in your cluster.</b>
 
+### I can see my service instance in SAP BTP, but not its corresponding custom resource in a cluster. What can I do?
```

Comment:
### I can see my service instance in SAP BTP, but not its corresponding custom resource my cluster. How can I restore it in the cluster?

Reply:
@kerenlahav  I think Avi wanted a more generally-worded question, but I changed.